src/components/Link/Link.scss

```diff
@@ -8,6 +8,7 @@
   background: none;
   border: 0;
   font-size: inherit;
+  font-weight: inherit;
```

Comment:
I also fixed this because I noticed our font-weights were not getting inherited on links to match the text they are a part of.